docs/product-getting-started-guide/src/main/asciidoc/task_deploying-application.adoc

```diff
@@ -19,9 +19,9 @@ To deploy an application in {PRODUCT}:
 . Log into Business Central.
 . Click *Menu* -> *Projects*.
 . Click on a project you want to deploy, for example *process-app-start*.
-. Click *Build & Deploy*. 
+. Click *Build & Deploy*.
 +
 * If there is no container with the project name, a container with default values is automatically created.
-* If there already is a container with the project name, a dialogue appears. You can change the name of your container and its alias. When finished, click *Ok*.
+* If there already is a container with the project name, a window opens. You can change the name of your container and its alias. When finished, click *Ok*.
```

Comment:
"If there already is a container with the project name, a window opens". Suggest: "A window opens if a container with the project name already exists."? WDYT?